### PR TITLE
Migrate to eslint-config-formidable

### PR DESCRIPTION
**Pull request by tptee**
Yay!

### CODE REVIEW

File: config/webpack/webpack.config.dev.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-var webpack = require('webpack');
-var config = require('./webpack.config');
+const webpack = require('webpack');
+const config = require('./webpack.config');
```
Comment (divmain) [inline config/webpack/webpack.config.dev.js]: It may be worth adding an `engines` entry in `package.json` since this won't work in Node  < 4.

**Comment by tptee** [comment]
I accidentally put some `devDependencies` into `dependencies` for the upcoming demo PR 😬 Will fix

File: src/store-enhancer.js [hunk 2]
```diff
@@ -174,26 +174,26 @@ export default ({
...
-          return null;
-        case GO_FORWARD:
-          history.goForward();
-          return null;
-        default:
-          // We return the result of dispatch here
-          // to retain compatibility with enhancers
-          // that return a promise from dispatch.
-          return store.dispatch(action);
+      case PUSH:
```
Comment (divmain) [inline src/store-enhancer.js]: Oh, this is strange.  Why isn't `case` indented?

Comment (tptee) [inline src/store-enhancer.js]: It's weirdly the rule in `eslint-config-formidable`...we had an override before, maybe we rethink this rule there?

File: config/webpack/webpack.config.dev.js (hunk 1 above)
Comment (tptee) [inline config/webpack/webpack.config.dev.js]: 💯

File: src/store-enhancer.js (hunk 2 above)
Comment (divmain) [inline src/store-enhancer.js]: I prefer the indentation, but its 100% up to you.  I'm surprised that's the default tho!

**Pull request merged**